walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,67 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    ArrayList<TimeRange> busyIntervals = new ArrayList<TimeRange>();
+    ArrayList<TimeRange> freeIntervals = new ArrayList<TimeRange>();
+    ArrayList<TimeRange> importantRanges = new ArrayList<TimeRange>();
+    
+    for(String attendee : request.getAttendees()){
```

Comment:
Please add some comments as the meanings of these variables are not straightforward. Worth stating what each is used for.